util/OptionsDB.cpp

```diff
@@ -431,6 +431,24 @@ void OptionsDB::SetFromCommandLine(const std::vector<std::string>& args) {
     }
 }
 
+void OptionsDB::SetFromFile(const boost::filesystem::path& file_path,
+                            const std::string& version) {
```

Comment:
After you left your initial comment, I figured out what you just said.
I think it would help newer contributors to add those three sentences to the Programming wiki page.

This particular file didn't have other multi-line definitions, so I didn't know about that style convention until you mentioned it.
(The one exception being the declaration with a long initializer list, but my first guess was new line for { after initializer lists, not because it was multi-line.)